clients/android/app/src/main/java/app/lockbook/loggedin/listfiles/FileModel.kt

```diff
@@ -27,28 +29,31 @@ class FileModel(path: String) {
     val errorHasOccurred: LiveData<String>
         get() = _errorHasOccurred
 
+    val unexpectedErrorHasOccurred: LiveData<String>
+        get() = _unexpectedErrorHasOccurred
+
     fun syncWorkAvailable(): Boolean {
         when (val syncWorkResult = CoreModel.calculateFileSyncWork(config)) {
             is Ok -> return true
             is Err -> when (val error = syncWorkResult.error) {
-                is CoreError.NoAccount -> {
+                is CalculateWorkError.NoAccount -> {
                     Timber.e("No account.")
                     _errorHasOccurred.postValue("Error! No account!")
                 }
-                is CoreError.CouldNotReachServer -> {
+                is CalculateWorkError.CouldNotReachServer -> {
                     Timber.e("Could not reach server despite being online.")
                     _errorHasOccurred.postValue(
-                        UNEXPECTED_ERROR_OCCURRED
+                        UNEXPECTED_CLIENT_ERROR
```

Comment:
Shouldn't be using that constant.

Reply:
Get rid of internet checks, just let them know they have no internet when they try to do something that does.